### DISCUSSION TITLE
Buenas otra pregunta

### DISCUSSION THREAD

**Discussion by mafer23** [Launch-X-Latam/MisionFrontEnd#458] [Team Posts]
¿Qué días son las transmisiones y hora?, que nunca he alcanzado a una y quisiera hacer preguntas en ese espacio porque por correo se demora un poco.

**Comment by CarlNauj** [comment]
Buenas tardes: Para ver los videos del curso, tiene que ingresar a Microsoft Teams (si no lo tiene lo puede descargar) e ingresar con su cuenta de Innovación.Espero poder haberle ayudado de algo.El 20 oct. 2022 13:19, Maria Fernanda Palencia ***@***.***> escribió:

¿Qué días son las transmisiones y hora?, que nunca he alcanzado a una y quisiera hacer preguntas en ese espacio porque por correo se demora un poco.

**Comment by doguedogue** [comment]
![Calendario Sesiones Launch X Latam](https://user-images.githubusercontent.com/23409026/197007332-f6ef7c1b-9115-440f-8180-7267b2c4af28.png)

**Comment by mafer23** [comment]
gracias porque la verdad van en 7 sesiones y nunca alcanzo a estar .
